python/gui/raster/qgsrasterminmaxwidget.sip

```diff
@@ -11,6 +11,9 @@ class QgsRasterMinMaxWidget: QWidget
 
     void setBands( const QList<int> & theBands );
 
+    QgsRectangle extent();
```

Comment:
The sip bindings also need the doxygen comments - it makes it easier to diff them against the header files.